content/docs/developer/anteaterapi/migrating.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Migrating from PeterPortal API
+---
+
+import { Steps, Step } from "fumadocs-ui/components/steps";
+
+PeterPortal API will be sunset **on Friday, January 31st, 2025, at 11:59 pm Pacific Standard Time**. After this time, requests made to either version of PeterPortal API will no longer be served.
+
+Anteater API is the successor to PeterPortal API. The sole upgrade path from either version of PeterPortal API is to Anteater API.
+
+We are happy to assist developers in the migration process. If you would like assistance, or have any questions regarding the migration, please contact us at [icssc@uci.edu](mailto:icssc@uci.edu). Please note however that our resources are limited, and as a result we may not be able to respond to all such requests.
```

Comment:
"As an organization founded on open source, we will prioritize requests based on their usage and licensing."